src/navigation/tabs/TabContainer.tsx

```diff
@@ -8,12 +8,12 @@ type PropsWithMoreParams<P = unknown> = P & {
   removeMarginTop?: boolean;
 };
 
-const TabContainer: React.FC<PropsWithMoreParams> = ({children}) => {
+const TabContainer: React.FC<PropsWithMoreParams> = ({children, removeMarginTop}) => {
```

Comment:
Rather than adding a new prop here, would it be simpler to place the following line at the top of the `TabContainer` component?:

`  const showArchaxBanner = useAppSelector(({APP}) => APP.showArchaxBanner);
`

We could then use the `showArchaxBanner` variable to conditionally show the top insets on android and then we wouldn't have to modify the 4 top-level tab files at all and can eliminate all instances of:
`<TabContainer removeMarginTop={showArchaxBanner ? true : false}>`